meja/meja.ml

```diff
@@ -126,6 +131,7 @@ let main =
           file := Some filename )
     usage_text ;
   let env = Initial_env.env in
+  Printexc.record_backtrace !exn_backtraces ;
```

Comment:
should recording happen even when `--compiler-backtraces` is not enabled?

Reply:
The `record_backtrace` takes a `bool` that sets recording on or off. `exn_backtraces` is `false` if the flag isn't passed, so it should only record with the flag

Reply:
oh, of course, thanks